front/src/utils/date.ts

```diff
@@ -75,6 +86,13 @@ export const formatToIsoDate = (date: number | string, formatDate: boolean = fal
   return dayjs.tz(date, userTimeZone).format(format);
 };
 
+/**
+ * Transform a milliseconds locale date to an ISO 8601 date
+ * @param msDate milliseconds date (elapsed from January 1st 1970)
+ * @return an ISO 8601 date (e.g. 2024-04-25T08:30:00+02:00)
+ */
+export const formatLocaleDateToIsoDate = (msDate: number | string) => dayjs.tz(msDate).format();
```

Comment:
It would better if we use `Date` instead of number/string as parameter. By passing a `Date`, we are sure it's a date that's passed, but with a `number/string`, you can't be sure.